### PR TITLE
Add expands functionality

### PR DESCRIPTION
**Pull request by stephaniehobson**
- add expands functionality
- add expands functionality to hotel suggestions
- change session summaries to use expands functionality
- rename and improve parentByClass
- start tracking buttons
- stop tracking speaker expands twice
- alphabetize style includes
- remove errant console log
- remove old hide/show functionality

### CODE REVIEW

**Comment by stephaniehobson** [comment]
@hoosteeno this is some fairly major functionality changes, I could really use a second set of eyes. Could you please test functionality without merging? I'm still testing too.

File: source/js/02_analytics.js [hunk 1]
```diff
@@ -66,6 +66,21 @@
...
+                // utility functions
+                function controlClasses(controls){
+                    var controlClasses = controls.className;
+                    // remove common utility class
+                    controlClasses = controlClasses.replace('section','');
+                    // make double spaces single
+                    controlClasses = controlClasses.replace(/\s{2,}/,'');
+                    // remove spaces at start and finish
+                    controlClasses = controlClasses.replace(/^\s/,'');
+                    controlClasses = controlClasses.replace(/\s$/,'');
```
Comment (hoosteeno) [inline source/js/02_analytics.js]: Not trim()?

File: source/js/02_analytics.js [hunk 2]
```diff
@@ -173,6 +190,37 @@
                     } else {
                         analytics.trackEvent(data);
                     }
+                } else if(trigger == 'BUTTON' || parent == 'BUTTON') {
```
Comment (hoosteeno) [inline source/js/02_analytics.js]: Should we use === here?

File: source/js/02_analytics.js [hunk 3]
```diff
@@ -173,6 +190,37 @@
...
+                    var category;
+                    var action;
+                    var label;
+                    var button = (trigger == 'BUTTON') ? event.target : event.target.parentNode;
```
Comment (hoosteeno) [inline source/js/02_analytics.js]: Should we use === here?

File: source/js/expand.js [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+    /*
+        Basic expand/collapse utilities
+    */
+
+    var isHash = false;
+    var targetId = null;
+    if(window.location.hash) {
+        isHash = true;
+        targetId = window.location.hash;
+        targetId = targetId.replace("#", "");
```
Comment (hoosteeno) [inline source/js/expand.js]: Nits! replace " with '

**Pull request merged**